Framework/TestHelpers/inc/MantidTestHelpers/FileComparisonHelper.h

```diff
@@ -1,6 +1,7 @@
 #ifndef FILECOMPARISONHELPER_H_
 #define FILECOMPARISONHELPER_H_
 
+#include <iterator>
```

Comment:
As there were no other changes in this file is it necessary to add this to the header?